distributed/client.py

```diff
@@ -486,13 +488,29 @@ def __init__(self, address=None, loop=None, timeout=5,
         super(Client, self).__init__(connection_args=self.connection_args,
                                      io_loop=self.loop)
 
-        self.start(timeout=timeout, asynchronous=asynchronous)
+        self.start(timeout=timeout)
 
-        from distributed.channels import ChannelClient
-        ChannelClient(self)  # registers itself on construction
         from distributed.recreate_exceptions import ReplayExceptionClient
         ReplayExceptionClient(self)
 
+    @property
+    def asynchronous(self):
```

Comment:
I think either the name of this property and/or its semantics and/or its docstring should be changed, since all three seem to be saying different things :-)